lsif-java/src/main/scala/com/sourcegraph/lsif_java/commands/SnapshotLsifCommand.scala

```diff
@@ -209,56 +217,54 @@ object SnapshotLsifCommand {
       val edges = ListBuffer.empty[(String, String)]
       val inputs = mutable.Map.empty[String, Input]
       def input(range: LsifObject): Input = {
-        val uri = G
-          .predecessors(range)
-          .asScala
-          .iterator
-          .filter(_.getLabel() == "document")
-          .next()
-          .getUri()
+        val uri =
+          G.predecessors(range)
+            .asScala
+            .iterator
+            .filter(_.getLabel == "document")
+            .next()
+            .getUri
         inputs.getOrElseUpdate(uri, Input.path(Paths.get(URI.create(uri))))
       }
       def renderPos(pos: LsifPosition): String = {
-        s"${pos.getLine()}:${pos.getCharacter()}"
+        s"${pos.getLine}:${pos.getCharacter}"
       }
       val addedVertexes = mutable.Set.empty[String]
       def render(node: LsifObject): String = {
-        (node.getType(), node.getLabel()) match {
+        (node.getType, node.getLabel) match {
```

Comment:
You can disable the IntelliJ warning to remove the `()`. I think it's annoying that IntelliJ enables that warning by default. Maybe I should just change Metals to no insert the parentheses by default when calling Java methods. Anyways, no need to change this PR, I'm OK with removing the parentheses